pkg/api/admin/openshiftclustercredentials_convert.go

```diff
@@ -0,0 +1,16 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+type openShiftClusterCredentialsConverter struct{}
+
+// ToExternal is supposed to return a struct with admin username and password,
+// but we do not expose secrets in the admin API, so we just return an empty struct
+func (*openShiftClusterCredentialsConverter) ToExternal(*api.OpenShiftCluster) interface{} {
+	return struct{}{}
```

Comment:
This means that post to list credentials will receve an empty json object. I don't think that we will have any credentials that we will want to list here in the future. Probably we should look into disabling the list credentials route for the admin API completely?

Reply:
Agree, perhaps `OpenShiftClusterCredentialsConverter()` can return nil for this API and we can check for that in openshiftclustercredentials_post.go?

Reply:
I created a separate PR https://github.com/Azure/ARO-RP/pull/124 for this.